tweets/2022/03/09-registration-status.tweet

```diff
@@ -0,0 +1 @@
+8 instructors, 16 exercise leads, and 194 learners have already registered for the upcoming CodeRefinery workshop. We are looking for more exercise leads and you can still register as team, or for live stream: https://coderefinery.org/2022-03-22-workshop/
```

Comment:
```suggestion
8 instructors, 16 exercise leads, and 194 learners from 13 countries have already registered for the upcoming CodeRefinery workshop. We are looking for more exercise leads and you can still register as team, or for live stream: https://coderefinery.org/2022-03-22-workshop/
```

Reply:
13 countries!? 🎉 that's fantastic

Reply:
random spin-off idea, on the first workshop day (or after) might be cool to have a Europe (world?) map showing where from the participants are from, it's at least great to show off and argue about impact

Reply:
It's a great idea.

Reply:
If somebody has the time to create the map, I can provide then an up-to-date list of countries. I already know that I won't have time to create the map since there will be a lot for me to juggle.

Reply:
Yes, i could do that 👍